app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/text_input_govuk.scala.html

```diff
@@ -79,6 +79,7 @@ <h1 class="govuk-label-wrapper">
     <span id="@id" class="govuk-hint">
       @text
     </span>
+    <br/>
```

Comment:
Can you please check why this <br> is required? if possible please remove it.

Reply:
You could change the span wrapping the hint to a div to achieve the same thing.